index.html

```diff
@@ -0,0 +1,110 @@
+		<h2 class="main-text">
+			Your Perfect Tour in Uzbekistan Starts Here!
+		</h2>
+		<p class="text">
+			If you are planning a trip to Uzbekistan, Lorem ipsum is the best choice.Just specify the region you
+			prefer
+			to visit and one of our expert travel guides will provide you with a well-thought-out vacation plan. Book
+			your trip and enjoy the amazing experience!
+		</p>
+	</section>
+
+	<section class="section_2" id="section_2">
+		<h2 class="text-2">Main benefits</h2>
+		<div class="line-separator"></div>
+		<div class="section2-box">
+			<div class="box">
+				<img class="box-icons" src="./assets/images/icons/map_white_48dp.svg" alt="">
+				<h5>What we offer</h5>
+				<p class="box-text">We offer hundreds of sightseeing tours and custom packages which can meet any
+					taste
+					and budget.
+				</p>
+			</div>
+			<div class="box">
+				<img class="box-icons" src="./assets/images/icons/verified_white_48dp.svg" alt="">
+				<h5>Our experience</h5>
+				<p class="box-text">Each of our travel specialists meets first-class industry standards and has at
+					least
+					4.5 out of 5-star
+					rating.
+				</p>
+			</div>
+			<div class="box">
+				<img class="box-icons" src="./assets/images/icons/thumb_up_alt_white_48dp.svg" alt="">
+				<h5>Satisfaction guaranteed</h5>
+				<p class="box-text">Our experienced travel agents guarantee each client the most enjoyable and
+					memorable
+					experience.
+				</p>
+			</div>
+		</div>
+		<button type="button" class="btn">Book a tour</button>
+		<h2 class="all-benefits">SEE THE WHOLE BENEFITS</h2>
+	</section>
+
+	<section class="section_3" id="section_3">
+		<h2 class="text-3">Popular tours</h2>
+		<div class="line-separator"></div>
+		<div class="tours-box myToursSection"></div>
+		<div class="line-separator2"></div>
+		<div id="more-btn" class="more-btn">
+			<h6 id="more-btn-text" class="more-btn-text">MORE</h6>
+			<img id="more-icon" class="more-icon" src="./assets/images/icons/arrow-down.png" alt="">
+		</div>
+	</section>
+
+	<section id="section_4" class="section_4">
+		<h2 class="text-4">Partners</h2>
+		<div class="line-separator"></div>
+		<ul class="partners-list">
+			<li class="partners"><img class="partners-img1" src="./assets/images/partners/NBU.png" alt=""></li>
+			<li class="partners"><img class="partners-img2" src="./assets/images/partners/Advantour.png" alt=""></li>
+			<li class="partners"><img class="partners-img3" src="./assets/images/partners/YandexTaxi.png" alt="">
+			</li>
+			<li class="partners"><img class="partners-img4" src="./assets/images/partners/click.png" alt=""></li>
+		</ul>
+	</section>
+	<footer>
+		<img class="logo-img2" src="./assets/images/logo.svg" alt="">
+		<div class="footer-text">
+			<h5 class="footer-logo">Lorem Ipsum</h5>
+			<p class="footer-info">Phone:+123456789</p>
+			<p class="footer-info">All rights reserved.</p>
+		</div>
+	</footer>
+	<script src="./assets/JS/script.js"></script>
+</body>
```

Comment:
- [ ] On the about page, the footer seems to be broken, and your page allows the user to scroll left and go beyond on the page, kindly fix this by including `scroll-x: hidden.` for more check the image below.
![foteer1](https://user-images.githubusercontent.com/55635977/128428145-1006cec5-005b-43c8-9195-eb4d9cf042cf.png)

#### Fix that issue on different pages.

Reply:
This issue is well resolved! ✔️